packages/components/psammead-most-read/README.md

```diff
@@ -23,7 +23,7 @@ The `MostRead` component is designed to display the most read articles given a d
 <!-- prettier-ignore -->
 | Argument | Type | Required | Default | Example |
 | -------- | ---- | -------- | ------- | ------- |
-| items | array | yes | N/A | `[{ title: 'This is a headline', href: 'https://www.bbc.com' }]`
+| items | array | yes | N/A | `[{ id: 1, title: 'This is a headline', href: 'https://www.bbc.com', timestamp: null }]` |
```

Comment:
If we expect `null` values, can we have this as an optional prop instead and default it to `null`

Reply:
The reason I've added it as an optional prop here is that we can have 10 items in our array so there could possibly be 10 timestamp values and we would like to display the timestamp for that item if it is greater than 60days (I plan on having this 60days logic in Simorgh and passing the timestamp container a prop here 👀). Because the timestamp is tied to the item, I think to have it as a separate prop will make difficult to handle.

Reply:
thanks @thekp makes sense